#860 - Add code splitting

Manice18 opened the pull request:
<!-- If your PR fixes an open issue, use `Closes #435` to link your PR with the issue. #435 stands for the issue number you are fixing -->

## Fixes Issue 

<!-- Remove this section if not applicable -->

<!-- Example: Closes #31 -->
Closes #860 

## Changes proposed
Previously :- 
![image](https://github.com/MonalikaPatnaik/UMatter/assets/91601706/46297fd4-44b3-4ee9-84af-bbd1bf2cf827)
Now (after adding lazy loading) :- 
![image](https://github.com/MonalikaPatnaik/UMatter/assets/91601706/313dabe5-4ddb-442e-8903-850ae65b9470)
 

## Check List (Check all the applicable boxes) <!-- Follow the above conventions to check the box -->

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.

MonalikaPatnaik commented on public/index.html:
```diff
@@ -1,16 +1,6 @@
-        z-index: 999999;
-      }
-      </style>
     <meta charset="utf-8" />
```
Why you have removed the code for preloader?

MonalikaPatnaik commented on public/index.html:
```diff
@@ -30,20 +20,12 @@
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
     <script type="text/javascript"> window.$crisp=[];window.CRISP_WEBSITE_ID="e79efdd2-abee-4a1e-b868-c7929585ebd9";(function(){ d=document;s=d.createElement("script"); s.src="https://client.crisp.chat/l.js"; s.async=1;d.getElementsByTagName("head")[0].appendChild(s);})(); </script>
   </head>
-  <body class="dark-theme" onload="myFunction()">
```
why you have removed "onload="myFunction()"

MonalikaPatnaik commented:
It is giving error in deployment..
![image](https://github.com/MonalikaPatnaik/UMatter/assets/99353300/29f4a61c-8b18-4262-abf9-4619d357b776)

Manice18 commented:
Oh i am sorry it would be 'loader.css' and not 'Loader.css' 
I removed the preloader because it was clashing with the Suspense fallback loader for the home page, hence I added the same loader to all the pages

Manice18 commented:
@MonalikaPatnaik I will resolve all the conflicts and then add a PR
